### PR TITLE
Hero-main-index

### PR DESCRIPTION
**Pull request by LaserHuy**
hero and main section

### CODE REVIEW

**Review by yazidsef — APPROVED** [review]
ok

File: src/View/Home/index.html.twig [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+        <link rel="stylesheet" href="/assets/css/home/formation.css">
+        <link rel="stylesheet" href="/assets/css/home/card-dis.css">
+        <link rel="stylesheet" href="/assets/css/home/intro-main.css">
+    {% endblock %}
 {% block content %}
+    {% include 'components/home/hero.html.twig' %}
+    {% include 'components/home/formation.html.twig' %}
+    {% include 'components/home/intro-main.html.twig' %}
+
     <h1>Welcome to Oxygen School !</h1>
```
Comment (Benja-san) [inline src/View/Home/index.html.twig:15]: does this H1 still have its place here ? you already have one in your "hero" component

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/Home/index.html.twig (hunk 1 above)
Comment (LaserHuy) [reply src/View/Home/index.html.twig:15]: I will remove this line

File: src/View/Home/index.html.twig [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
     <p>Find a complete CRUD example on <a href="/items">/items</a> page.</p>
```
[the remark this answers is not in the document]
Comment (LaserHuy) [reply src/View/Home/index.html.twig:16]: ok it has been removed

**Review by Benja-san — APPROVED** [review]

**Pull request merged**